interface/patient_file/summary/demographics.php

```diff
@@ -231,6 +241,35 @@ function toggleIndicator(target,div) {
     }
 }
 
+// edit prescriptions dialog. @todo pull fancybox dependencies for this replace.
+// replace fancybox dialog sjp 11/25/17
+// called from stats.php.
+//
+function editScripts(url) {
+
+    var AddScript = function () {
+        var iam = top.tab_mode ? top.frames.editScripts : window[0];
+        iam.location.href = "<?php echo $GLOBALS['webroot']?>/controller.php?prescription&edit&id=&pid=<?php echo attr($pid);?>"
+    };
+    var ListScripts = function () {
+        var iam = top.tab_mode ? top.frames.editScripts : window[0];
+        iam.location.href = "<?php echo $GLOBALS['webroot']?>/controller.php?prescription&list&id=<?php echo attr($pid); ?>"
+    };
+    var title = '<?php echo xla('Prescriptions Add/Edit'); ?>';
+
+    oeModal(url, 'editScripts', 'modal-mlg', 300, title, {
+        buttons: [
+            {text: '<?php echo xla('Add'); ?>', close: false, style: 'primary  btn-sm', click: AddScript},
+            {text: '<?php echo xla('List'); ?>', close: false, style: 'primary  btn-sm', click: ListScripts},
+            {text: '<?php echo xla('Done'); ?>', close: true, style: 'success btn-sm', click: refreshme}
```

Comment:
Will be good to get @robertdown 's take on the styling of buttons in the popups. @robertdown , note you can test this on the up for grabs demo at:
http://www.open-emr.org/wiki/index.php/Development_Demo#Lambda_-_Up_For_Grabs_Demo

Reply:
No worries on buttons. Just haven't figured out if I want to embed the style classes or do an add class option.